src/Core/Data/RecipeReview.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace mbsoft.BrewClub.Data
+{
+	public class RecipeReview
+	{
+
+		public UserProfile Author { get; set; }
+
+		public string Comments { get; set; }
+		
+		public bool AuthorBrewedIt { get; set; }
+
+		public bool? AuthorLikedIt { get; set; }
```

Comment:
What are your thoughts on having a rating vs a boolean?  For example someone could give it 3 out of 5 stars.  We could use those ratings to calculate an average brewed rating for a recipe.